test/steps/kogitoapp.go

```diff
@@ -77,7 +77,7 @@ func (data *Data) deployServiceFromExampleFile(runtimeType, exampleFile string)
 
 // DeploymentConfig steps
 func (data *Data) kogitoApplicationHasPodsRunningWithinMinutes(dcName string, podNb, timeoutInMin int) error {
-	return framework.WaitForDeploymentConfigRunning(data.Namespace, dcName, podNb, timeoutInMin)
+	return framework.WaitForDeploymentRunning(data.Namespace, dcName, podNb, timeoutInMin)
```

Comment:
There is no DeploymentConfig in Kubernetes, do you see any problems if using the Deployment type when running in Kubernetes and Openshift?